### PR TITLE
Hod/mitre layout

### PR DESCRIPTION
**Pull request by hod-alpert**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/23383

## Description
Edited the MITRE Att&ck layout
## Screenshots
image

## Minimum version of Demisto
- [ ] 5.0.0
- [x] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
   - [x] No

### CODE REVIEW

**Comment by Itay4** [comment]
@hod-alpert any reason for this to still be in draft?

File: Packs/FeedMitreAttack/IndicatorFields/incidentfield-mitrecreated.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+ "id": "indicator_mitrecreated",
+ "version": -1,
+ "modified": "2020-09-13T16:40:18.932262+03:00",
+ "name": "Created (technique)",
```
Comment (Itay4) [inline Packs/FeedMitreAttack/IndicatorFields/incidentfield-mitrecreated.json:5]: why do we have `technique` here? also lets add `MITRE` here so this wont be too generic

File: Packs/FeedMitreAttack/IndicatorFields/incidentfield-mitremodified.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+ "id": "indicator_mitremodified",
+ "version": -1,
+ "modified": "2020-09-13T16:40:46.057661+03:00",
+ "name": "Modified (technique)",
```
Comment (Itay4) [inline Packs/FeedMitreAttack/IndicatorFields/incidentfield-mitremodified.json:5]: why do we have `technique` here? also lets add `MITRE` here so this wont be too generic

File: Packs/FeedMitreAttack/ReleaseNotes/1_1_2.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+
+#### Integrations
+##### MITRE ATT&CK Feed
+- Changed indicator fields
```
Comment (Itay4) [inline Packs/FeedMitreAttack/ReleaseNotes/1_1_2.md:4]: add here that docker image was updated (we have a standard generic line for it)

File: Packs/FeedMitreAttack/Layouts/layoutscontainer-MITRE_ATT&CK.json [hunk 4]
```diff
@@ -1,541 +1,853 @@
...
-                        "w": 1,
-                        "x": 1,
-                        "y": 5
-                    }
-                ],
-                "type": "custom"
-            }
-        ]
-    }
+ "id": "MITRE ATT\u0026CK",
```
Comment (Itay4) [inline Packs/FeedMitreAttack/Layouts/layoutscontainer-MITRE_ATT&CK.json:2]: lets keep the id and name as they were before

File: Packs/FeedMitreAttack/Layouts/layoutscontainer-MITRE_ATT&CK.json [hunk 5]
```diff
@@ -1,541 +1,853 @@
...
+        "index": 2,
+        "name": "Add tags",
+        "scriptId": "Builtin|||appendIndicatorField",
+        "sectionItemType": "button",
+        "startCol": 0
+       },
+       {
+        "args": {
+         "field": {
+          "simple": "Tags"
```
Comment (Itay4) [inline Packs/FeedMitreAttack/Layouts/layoutscontainer-MITRE_ATT&CK.json:170]: i think this should be `tags`

File: Packs/FeedMitreAttack/Layouts/layoutscontainer-MITRE_ATT&CK.json [hunk 6]
```diff
@@ -1,541 +1,853 @@
...
+        "height": 53,
+        "id": "18d844d0-d1af-11ea-a7f6-69bb41e2a387",
+        "index": 1,
+        "sectionItemType": "field",
+        "startCol": 0
+       },
+       {
+        "args": {
+         "field": {
+          "simple": "Tags"
```
Comment (Itay4) [inline Packs/FeedMitreAttack/Layouts/layoutscontainer-MITRE_ATT&CK.json:154]: i think this should be `tags`

File: Packs/FeedMitreAttack/Integrations/FeedMitreAttack/FeedMitreAttack.py [hunk 7]
```diff
@@ -1,12 +1,11 @@
 import demistomock as demisto
 from CommonServerPython import *
-from CommonServerUserPython import *
 
 from typing import List, Dict, Set
 import json
 import requests
 from stix2 import TAXIICollectionSource, Filter
-from taxii2client import Server, Collection, ApiRoot
+from taxii2client.v20 import Server, Collection, ApiRoot
```
Comment (Itay4) [inline Packs/FeedMitreAttack/Integrations/FeedMitreAttack/FeedMitreAttack.py:8]: make sure you're synced with https://github.com/demisto/etc/issues/26353 with these change

**Review by Itay4 — CHANGES_REQUESTED** [review]
lets go over this together

File: Packs/FeedMitreAttack/Integrations/FeedMitreAttack/FeedMitreAttack.py (hunk 7 above)
Comment (hod-alpert) [reply Packs/FeedMitreAttack/Integrations/FeedMitreAttack/FeedMitreAttack.py:8]: Synced, He implemented the same solution

File: Packs/FeedMitreAttack/Scripts/ShowMITREName/ShowMITREName.py [hunk 8]
```diff
@@ -0,0 +1,12 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+mitre_name = demisto.get(demisto.args()['indicator'], 'CustomFields.mitrename')
+err_msg = "MITRE Name was not found"
+
+if not mitre_name:
```
Comment (Itay4) [inline Packs/FeedMitreAttack/Scripts/ShowMITREName/ShowMITREName.py:8]: remove error message

File: Packs/FeedMitreAttack/Scripts/ShowMITREID/ShowMITREID.py [hunk 9]
```diff
@@ -0,0 +1,12 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+mitre_id = demisto.get(demisto.args()['indicator'], 'CustomFields.mitreid')
+err_msg = "MITRE ID was not found"
+
+if not mitre_id:
```
Comment (Itay4) [inline Packs/FeedMitreAttack/Scripts/ShowMITREID/ShowMITREID.py:8]: remove return_error

File: Packs/FeedMitreAttack/Scripts/ShowMITREID/ShowMITREID.yml [hunk 10]
```diff
@@ -0,0 +1,18 @@
+commonfields:
+  id: ShowMITREID
+  version: -1
+name: ShowMITREID
```
Comment (Itay4) [inline Packs/FeedMitreAttack/Scripts/ShowMITREID/ShowMITREID.yml:4]: MitreIDDynamicSection

File: Packs/FeedMitreAttack/Scripts/ShowMITREName/ShowMITREName.yml [hunk 11]
```diff
@@ -0,0 +1,18 @@
+commonfields:
+  id: ShowMITREName
+  version: -1
+name: ShowMITREName
```
Comment (Itay4) [inline Packs/FeedMitreAttack/Scripts/ShowMITREName/ShowMITREName.yml:4]:
```suggestion
name: MitreNameLayoutDynamicSection
```

File: Packs/FeedMitreAttack/Scripts/ShowMITREID/ShowMITREID.yml [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+comment: Shows Mitre ID
```
Comment (Itay4) [inline Packs/FeedMitreAttack/Scripts/ShowMITREID/ShowMITREID.yml:5]: improve description

File: Packs/FeedMitreAttack/Scripts/ShowMITREName/ShowMITREName.yml [hunk 13]
```diff
@@ -0,0 +1,18 @@
...
+comment: Shows Mitre Name
```
Comment (Itay4) [inline Packs/FeedMitreAttack/Scripts/ShowMITREName/ShowMITREName.yml:5]: improve description

File: Packs/FeedMitreAttack/ReleaseNotes/1_1_2.md [hunk 14]
```diff
@@ -0,0 +1,16 @@
...
+- Updated docker image to the latest version.
+
+#### Layouts
+- **MITRE ATT&CK Indicator**
+
+#### Scripts
+##### ShowMITREID
+Created new script to show MITRE indicator ID
```
Comment (Itay4) [inline Packs/FeedMitreAttack/ReleaseNotes/1_1_2.md:12]: here should be the script desc

File: Packs/FeedMitreAttack/ReleaseNotes/1_1_2.md [hunk 15]
```diff
@@ -0,0 +1,16 @@
+
+#### Integrations
+##### MITRE ATT&CK Feed
+- Changed indicator fields
```
Comment (Itay4) [inline Packs/FeedMitreAttack/ReleaseNotes/1_1_2.md:4]: improve this rn

File: Packs/FeedMitreAttack/Integrations/FeedMitreAttack/FeedMitreAttack.py [hunk 16]
```diff
@@ -23,7 +22,6 @@
     "mitredetection": {"name": "x_mitre_detection", "type": "str"},
     "mitreextendedaliases": {"name": "x_mitre_aliases", "type": "list"},
     "mitreexternalreferences": {"name": "external_references", "type": "dict"},
-    "mitreid": {"name": "id", "type": "str"},
```
Comment (Itay4) [inline Packs/FeedMitreAttack/Integrations/FeedMitreAttack/FeedMitreAttack.py:26]: make sure we dont break bc

File: Packs/FeedMitreAttack/ReleaseNotes/1_1_2.md [hunk 17]
```diff
@@ -0,0 +1,16 @@
...
+- Updated docker image to the latest version.
+
+#### Layouts
+- **MITRE ATT&CK Indicator**
```
Comment (Itay4) [inline Packs/FeedMitreAttack/ReleaseNotes/1_1_2.md:8]: please add here rn

**Review by Itay4 — APPROVED** [review]

**Comment by kirbles19** [comment]
@hod-alpert Done.

**Pull request merged**